docs/build/html/_sources/er-force-sim/index.rst.txt

```diff
@@ -0,0 +1,43 @@
+ER-Force Simulation Installation
+================================
+
+Download ER-Force Sim
+----------------------
+
+First git clone the ER-Force Simulator from the following repo:
+
+.. code-block:: sh
+
+   git clone https://github.com/robotics-erlangen/framework.git
+
+Change directory into the recently cloned repo
+
+Building the framework
+----------------------
+
+Run the following code line by line
+
+.. code-block:: sh
+
+    mkdir build && cd build
+    cmake ..
+    make simulator-cli
+
+This builds an executable in framework/build/bin
+
+Running the Simulation
+----------------------
+
+Open a new terminal window and change directory into framework/build/bin
+
+In this folder, there should be a file named "simulator-cli"
+
+Run this file by doing the following
```

Comment:
you could just put this on the same line as the above sentence, since it'll be opened in a web browser that takes care of limiting the width for the user